Makefile

```diff
@@ -1,10 +1,22 @@
 up:
 	docker-compose -f dev.yml up -d;
-	./setup_pubsub.sh
+	./setup_dependencies.sh
 	
 down:
 	docker-compose -f dev.yml down
 
+docker:
+	docker build -t eu.gcr.io/census-rm-ci/census-rm-pubsub-adapter .
+
+build:
+	go build -race .
```

Comment:
do we need to build for specific/multiple architectures here? i.e. different binary for mac and linux (docker image)

Reply:
If we want to, I guess it future proofs us if we want to switch architecture of our docker image.

Reply:
This is ok for now, something to keep in mind